service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -82,19 +82,19 @@ void chainCallsTogether() throws Exception {
         """
         {
           "workflow_url" : "%s",
-          "workflow_param_definitions" : [ {
-            "parameter_name" : "myworkflow.mycall.inputname1",
-            "parameter_type" : "String",
+          "workflow_input_definitions" : [ {
```

Comment:
You probably want to add an output definition here too, to prove that the API definition (and thus JSON parsing) is all working properly.

Reply:
These should be all set! Checks were failing as a result of these being mismatched, but they're passing now.